README.md

```diff
@@ -0,0 +1,16 @@
+# Univ. of Cambridge Digital Hospitals webapp
+
+## Installation + build instructions
+
+```bash
```

Comment:
Instead of using the latest version of node.js, use
Node v18.12.0 (LTS) which is the long term support version. Better to redirect to the node.js website instead of using script which only works on unix.